mlir/include/mlir/Dialect/LLVMIR/LLVMOpBase.td

```diff
@@ -67,6 +67,9 @@ class LLVM_PointerInAddressSpace<int addressSpace> : Type<
     # addressSpace # ")";
 }
 
+// Type constraint accepting any LLVM pointer type in address space 0.
+def LLVM_Pointer : LLVM_PointerInAddressSpace<0>;
```

Comment:
That seems a bit odd. Shouldn't this have a name that indicates that it's in the default address space?

Reply:
I went for LLVM_DefaultPointer now.